app/renderer/components/tabs/content/audioTabIcon.js

```diff
@@ -29,71 +28,61 @@ class AudioTabIcon extends React.Component {
   }
 
   get audioIcon () {
-    const isNotMuted = this.props.pageCanPlayAudio && !this.props.audioMuted
-
-    return isNotMuted
+    return this.props.audioPlaying
       ? globalStyles.appIcons.volumeOn
       : globalStyles.appIcons.volumeOff
   }
 
   toggleMute (event) {
     event.stopPropagation()
-    windowActions.setAudioMuted(this.props.frameKey, this.props.tabId, !this.props.audioMuted)
+    windowActions
+      .setAudioMuted(this.props.frameKey, this.props.tabId, this.props.audioPlaying)
   }
 
   mergeProps (state, ownProps) {
     const currentWindow = state.get('currentWindow')
-
-    // AudioIcon will never be created if there is no frameKey, but for consistency
-    // across other components I added teh || Immutable.Map()
-    const frame = frameStateUtil.getFrameByKey(currentWindow, ownProps.frameKey) || Immutable.Map()
-    const breakpoint = ownProps.breakpoint
+    const tabId = ownProps.tabId
+    const frameKey = frameStateUtil.getFrameKeyByTabId(currentWindow, tabId)
 
     const props = {}
-    // used in other functions
-    props.frameKey = ownProps.frameKey
-    props.pageCanPlayAudio = !!frame.get('audioPlaybackActive')
-    props.tabId = frame.get('tabId', tabState.TAB_ID_NONE)
-    props.audioMuted = frame.get('audioMuted')
-    props.showAudioIcon = breakpoint === 'default' && !!frame.get('audioPlaybackActive')
+    props.frameKey = frameKey
+    props.showAudioIcon = audioState.showAudioIcon(currentWindow, frameKey)
+    props.audioPlaying = !audioState.isAudioMuted(currentWindow, frameKey)
+    props.canPlayAudio = audioState.canPlayAudio(currentWindow, frameKey)
+    props.isPinned = tabState.isTabPinned(state, tabId)
+    props.tabId = tabId
 
     return props
   }
 
   render () {
-    if (!this.props.showAudioIcon) {
+    if (this.props.isPinned || !this.props.showAudioIcon) {
       return null
     }
+
     return <TabIcon
-      className={css(styles.icon, styles.icon_audio)}
+      data-test-id={this.audioIcon}
+      className={css(styles.audio__tabIcon)}
```

Comment:
would you mind renaming this to `styles.audioTa__icon` following https://github.com/brave/browser-laptop/pull/10691/commits/d69c3f7fe2fc37eaee262cf4bcf9fe4bb259e08d#diff-08a3077ca08d52dfaad29de7ae4d904aR75 or renaming the other?

Reply:
Confirmed this was addressed on https://github.com/brave/browser-laptop/pull/10691/files#diff-1a5be0fc07ffeb6eda18603b57e9fce9R65

On the commit: https://github.com/brave/browser-laptop/commit/01092fe798c23ecd1c6d61c51c3e74e0bfe8d02b#diff-1a5be0fc07ffeb6eda18603b57e9fce9R65